Simplified/Audiobooks/LCP/LCPAudiobooks.swift

```diff
@@ -72,44 +79,17 @@ extension LCPAudiobooks: DRMDecryptor {
   ///   - resultUrl: URL to save decrypted file at.
   ///   - completion: decryptor callback with optional `Error`.
   func decrypt(url: URL, to resultUrl: URL, completion: @escaping (Error?) -> Void) {
-    loadLicense { [weak self] license, error in
-      guard let self = self else {
-        completion(nil)
-        return
-      }
-      if let error = error {
-        NYPLErrorLogger.logError(error, summary: "Error loading license for LCP audiobook")
-        completion(error)
-        return
-      }
-      guard let license = license else {
-        completion(nil)
-        return
-      }
+    let asset = FileAsset(url: self.audiobookUrl)
+    streamer.open(asset: asset, allowUserInteraction: false) { result in
       do {
-        let audiofileContent = try self.container.data(relativePath: url.path)
-        let data = try license.decipher(audiofileContent)
-        try data?.write(to: resultUrl, options: .atomic)
+        let publicatoin = try result.get()
+        let resource = publicatoin.get(url.path)
+        let data = try resource.read().get()
+        try data.write(to: resultUrl)
         completion(nil)
       } catch {
         NYPLErrorLogger.logError(error, summary: "Error decrypting LCP audio file \(url)")
-        completion(error)
-        return
-      }
-    }
-  }
-  
-  /// Load `DRMLicense` license for audiobook once
-  /// - Parameter completion: `LCPError`, if any
-  private func loadLicense(completion: @escaping (_ license: DRMLicense?, _ error: Error?) -> Void) {
-    lcpService.loadPublication(at: audiobookUrl, drm: DRM(brand: .lcp)) { result in
-      switch result {
-      case .success(let drm):
-        completion(drm?.license, nil)
-      case .failure(let error):
-        completion(nil, error)
-      case .cancelled:
-        completion(nil, nil)
+        completion(LCPAudiobooks.nsError(for: error))
```

Comment:
it is logged in SimplyE in Objective-C part of code, `NYPLBookCellDelegate.m`, in `- (void)registerCallbackForLogHandler`

Reply:
I meant why not simply calling `completion(error)` instead, since `error` is of type Error, which is the requirement for the `completion` signature if I understand

Reply:
replaced with `error`